src/olympia/api/urls.py

```diff
@@ -1,14 +1,19 @@
 from django.conf.urls import include, url
 
 
+v3_api_urls = [
+    url(r'^abuse/', include('olympia.abuse.urls')),
+    url(r'^accounts/', include('olympia.accounts.urls')),
+    url(r'^addons/', include('olympia.addons.api_urls')),
+    url(r'^', include('olympia.discovery.api_urls')),
+    url(r'^reviews/', include('olympia.ratings.api_urls')),
+    url(r'^reviewers/', include('olympia.reviewers.api_urls')),
+    url(r'^', include('olympia.signing.urls')),
+    url(r'^activity/', include('olympia.activity.urls')),
+    url(r'^github/', include('olympia.github.urls')),
+]
```

Comment:
I'm not sure we even want to prefix this as `v3`, **new** endpoints should easily be addable to existing endpoints so that a new `/stats/` endpoint will work for both, v3 and v4. The only thing we have to be careful about are changes to existing endpoints imho.

Reply:
hmm.  I had idea we'd just add new apis to v4 - gotta encourage people to upgrade!😆 
Petty forced-upgrading aside, I'm not sure adding anything to v3 is practical - if we're saying its stable and only has non-breaking changes then it's impossible to add an endpoint that's still under development because it will inherently be breaking and changing all the time.

Reply:
Hrm, I think I do agree. Yeah, I think that'll work for me for now :laughing: